### PR TITLE
Added installation for cds

### PR DESCRIPTION
**Pull request by ManuelB**
When running these samples for the first time it is necessary to install the cds binaries globally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,10 @@ cd samples
 In the samples folder run:
 
 ```sh
+# Download and install dependencies into node_modules folder
 npm install
+# install cds binaries globally
+npm i -g @sap/cds-dk
```
Comment (chgeo) [inline README.md:32]: Could you move this to the _Preliminaries_ section above?

**Comment by ManuelB** [comment]
@chgeo Done.

**Review by chgeo — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -26,6 +29,7 @@ cd samples
 In the samples folder run:
 
 ```sh
+# Download and install dependencies into node_modules folder
```
Comment (danjoa) [inline README.md:32]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -8,9 +8,12 @@ Find here a collection of samples for the [SAP Cloud Application Programming Mod
 ### Preliminaries
 
 1. [Install @sap/cds-dk](https://cap.cloud.sap/docs/get-started/) as documented in [capire](https://cap.cloud.sap)
+   ```sh
+   # install cds tools globally
```
Comment (danjoa) [inline README.md:12]:
```suggestion
```

**Comment by chgeo** [comment]
Thanks!

**Pull request merged**